### DISCUSSION TITLE
为什么使用官方模型文件,在OTB2015测试集上,mobilenetv2作为backbone的测试结果不如Alexnet?

### DISCUSSION THREAD

**Discussion by Dennisky** [STVIR/pysot#488] [Reproduce]
yaml文件和pth文件均使用官方提供的.这个实验结果与论文不符啊

**Comment by ZhiyuanChen** [comment]
使用的配置文件是什么呢？

**Reply by Dennisky** [reply]
官方google drive中siamrpn_mobilev2_l234_dwxcorr文件夹下面的yaml 文件和 pth 文件

**Reply by ZhiyuanChen** [reply]
> 官方google drive中siamrpn_mobilev2_l234_dwxcorr文件夹下面的yaml 文件和 pth 文件

因为他不是siamrpn_mobilev2_l234_dwxcorr_otb

**Reply by Dennisky** [reply]
您的意思是算法针对OTB数据集进行调参了，且官方没有提供这个版本的模型？

**Reply by ZhiyuanChen** [reply]
对

**Reply by Dennisky** [reply]
> 对

还有就是siamRPN++论文中图9提到mobilenet为backbone的速度快于siamRPN，实际测试不是这样啊，mobilenet的明显比原来alexnet为backbone慢的多啊
